src/wp-includes/l10n.php

```diff
@@ -1988,3 +1988,15 @@ function wp_get_word_count_type() {
 
 	return $wp_locale->get_word_count_type();
 }
+
+/**
+ * Returns a boolean to indicate whether a translation exists for a given string with optional text domain and locale.
+ *
+ * @param string $singular  Singular translation to check.
+ * @param string $textdomain  Optional. Text domain. Default 'default'.
+ * @param ?string $locale  Optional. Locale. Default current locale.
```

Comment:
As per the [coding standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/), these should all be aligned.

Let's also tentatively add `@since 6.7.0`

Reply:
Done